mobile/components/Card.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { StyleProp, ViewStyle, View} from "react-native";
+import { Card as TamaguiCard, CardProps as TamaguiCardProps} from 'tamagui'
+import { tokens } from "../theme/tokens";
+
+export interface CardProps extends TamaguiCardProps {
+    customStyle?: StyleProp<ViewStyle>
+    children? : React.ReactNode;
+}
+
+/**
+ * This component is a HOC over the Tamagui Button.
+ * @param {ButtonProps} props - The props for the `Buttom` component.
+ * @returns {JSX.Element} The rendered `Card` component.
+ */
```

Comment:
please remove these comments as they don't relate to the Card but to the Button

Reply:
done